pyne/material.pyx

```diff
@@ -618,6 +618,29 @@ cdef class _Material:
         return nucvec_proxy
 
 
+    def dose_per_g(self, dose_type, source=0):
+        """This provides the dose per gram using the comp of the the Material.
+        ext_air_dose returns mrem/h per g per m^3
+        ext_soil_dose returns mrem/h per g per m^2
+        ingest_dose returns mrem per g
+        inhale_dose returns mrem per g
```

Comment:
sorry, I thought I added a line comment here earlier. I was thinking these lines here should either be in the `nucvec` returns section or that should be updated to make it clear what you are returning.